packages/core/src/di/injectable.ts

```diff
@@ -70,8 +70,8 @@ export interface Injectable {
    * - 'root' : The application-level injector in most apps.
    * - 'platform' : A special singleton platform injector shared by all
    * applications on the page.
-   * - 'any' : Provides a unique instance in every module (including lazy modules) that injects the
-   * token.
+   * - 'any' : Provides a unique instance in every lazy loaded modules and all eager loaded module
```

Comment:
```suggestion
   * - 'any' : Provides a unique instance in each lazy loaded module while all eagerly loaded modules
```

Is this the definition? That it will give a unique instance for each lazy module, but that all eagerly loaded ones will share one? (I'm still learning this.)

Reply:
Yeah, this is what i wanted to write, thanks :)